src/components/10-atoms/input-text/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 6.2.1
+
+Remove safari hack to fix and obsolete code #2345
```

Comment:
grammar: to fix **what?**

Reply:
spelling: **S**afari

Reply:
grammar: 'to obsolete' is not an existing verb IMHO. Suggestion: ... and **remove** obsolete code (showMessage getter).